Examples/KlaviyoSwiftExamples/Shared/MenuView.swift

```diff
@@ -8,80 +8,55 @@ struct MenuView: View {
     @State private var showingCheckout = false
 
     var body: some View {
-        GeometryReader { _ in
-            VStack(spacing: 0) {
-                // Header
-                HStack {
+        NavigationView {
```

Comment:
```suggestion
        NavigationStack {
```
`NavigationView` is deprecated so we should use this instead. This change requires bumping our minimum iOS version to iOS 16.0, but I think it's fine to make that change